repose-aggregator/external/service-clients/auth/pom.xml

```diff
@@ -46,6 +46,12 @@
         </dependency>
 
         <dependency>
+            <groupId>org.openrepose</groupId>
```

Comment:
I could also change all references from `org.openrepose.core.filter.logic.FilterDirector.SC_TOO_MANY_REQUESTS` to `org.springframework.http.HttpStatus.TOO_MANY_REQUESTS.value()`.

That is probably the cleaner way.